test/libsolidity/syntaxTests/pragma/invalid_range_operator_range_second.sol

```diff
@@ -0,0 +1,3 @@
+pragma solidity 0.8.17 - >0.8.20;
```

Comment:
Please add a case where the range is prefixed with something that's not one of the currently allowed operators. The PR does not change that case but I don't think we have that case already covered.